humility-hiffy/src/lib.rs

```diff
@@ -884,7 +896,7 @@ impl<'a> HiffyContext<'a> {
                     }
                 })
             }
-            Err(e) => {
+            Err(IpcError::Error(e)) => {
```

Comment:
Can we use `op.strerror` here?

Reply:
Yes, I believe we can --- I didn't do that because this code didn't use it
previously. I'll change it to.